src/test/kotlin/minesweeper/inputdata/MineGameConfigTest.kt

```diff
@@ -0,0 +1,18 @@
+package minesweeper.inputdata
+
+import org.junit.jupiter.api.Test
+
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.DisplayName
+import java.lang.IllegalArgumentException
+
+internal class MineGameConfigTest {
+
+    @Test
+    @DisplayName("map 크기 보다 지뢰의 개수가 많을 수 없다.")
+    fun mapSizeCompare() {
+        assertThrows(IllegalArgumentException::class.java) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThrows<IllegalArgumentException> {
```